mkosi/__init__.py

```diff
@@ -3214,7 +3257,7 @@ def run_postinst_script(
             nspawn_params += nspawn_params_for_blockdev_access(args, loopdev)
 
         run_workspace_command(args, root, ["/root/postinst", verb],
-                              network=(args.with_network is True),
+                              network=(args.with_network is True and not args.use_mirror_verbatim),
```

Comment:
I wouldn't couple `--use-mirror-verbatim` to the network option. It's not because someone is using a local mirror that they will not want to access the network in the post install script.

Reply:
There was some side-effect when using with-network=no, let me check again

Reply:
ah yes, `-C` is passed to `dnf` which breaks it, maybe I can just add the workaround for that

Reply:
ok, I've changed this to just skip `-C` with a TODO to figure it out later, and remove the rest.